_config.yml

```diff
@@ -90,27 +90,27 @@ rockstarSpeakersButtonText: "See all speakers"
 # Location Block
 locationBlockTitle: "Location"
 eventLocationName: "Porto"
-eventLocationLink: "http://google.com"
-eventStreetAddress: "Rua dos Testers"
+eventLocationLink: "TBA"
+eventStreetAddress: "TBA"
 eventAddressLocality: "Porto"
 eventAddressLocalityRegion: "Porto"
-eventPostalCode: "1234-123"
+eventPostalCode: ""
 eventAddressCountry: "Portugal"
 eventStartTime: "2018-06-16T10:00"
 eventDoorTime: "09:00"
 eventEndTime: "2018-06-16T10:00"
-eventPlaceCoordinates: "49.843237, 24.028751"
-mapCenterCoordinates: "49.842537, 24.025701"
-mapMobileCenterCoordinates: "49.841620, 24.029411"
+eventPlaceCoordinates: "41.157944, -8.629107"
+mapCenterCoordinates: "41.157944, -8.629107"
+mapMobileCenterCoordinates: "41.157944, -8.629107"
 
 # Venue Block
 venueBlockTitle: "Venue"
 venueBigImage: "venue_0.jpg"
 venueSmallImage1: "venue_1.jpg"
```

Comment:
Can we put placeholders instead of this images?

Reply:
@FabioCB improved

![venue_after_fix](https://user-images.githubusercontent.com/11976836/38049324-d53668be-32bf-11e8-89f6-81594820c6c0.png)